### PR TITLE
Set validation attribute on fields

### PR DESCRIPTION
**Pull request by pxlrbt**
`Password::min(4)` is in the default rules so there should be a translation for that, too.

### CODE REVIEW

**Review by zepfietje — APPROVED** [review]
Looks ready to be merged to me, @danharrin.

File: packages/app/src/Pages/Auth/Register.php [hunk 1]
```diff
@@ -146,6 +146,7 @@ protected function getUserModel(): string
     protected function getMessages(): array
     {
         return [
+            'password.min' => __('validation.min.string', ['attribute' => __('filament::pages/auth/register.fields.password.validation_attribute')]),
```
Comment (danharrin) [inline packages/app/src/Pages/Auth/Register.php]: Shouldn't these be `data.password.min`? Or, can we not just set the `validationAttribute()` on the field?

Comment (zepfietje) [reply packages/app/src/Pages/Auth/Register.php]: Oh maybe yes actually, haven't tested.

**Comment by zepfietje** [comment]
Reworked this PR, @danharrin. ✅

**Pull request merged**